### PR TITLE
Active page underlining and navbar fixes

### PR DESCRIPTION
**Pull request by elucidium**
(navbar fixes include alignment and figuring out how to get the hamburger/x icon to display properly in the open state)

### CODE REVIEW

**Review by sabrinazhai — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
 
   <nav class="nav-left">
     <ul>
-      <li><a href="index.html">Home</a></li>
+      <li><a href="#" id="nav-active">Home</a></li>
```
Comment (mikinty) [inline index.html:78]: I think from the figma, we're not going to underline the home link in index

File: src/css/menu.css [hunk 2]
```diff
@@ -150,34 +160,40 @@
...
-  background-size: cover;
-  background-position: right center;
+  /*background-size: cover;
+  background-position: right center;*/
   transition: opacity 3s;
   opacity: 0;
 }
 
-.mobile-head .mobile-head-background.active {
+/*.mobile-head .mobile-head-background.active {
```
Comment (mikinty) [inline src/css/menu.css:194]: Can you just delete this instead of commenting out?

File: src/css/menu.css [hunk 3]
```diff
@@ -150,34 +160,40 @@
...
   top: 0;
   z-index: 999;
-  background-image: url(../../assets/img/bg_web_l.png);
+  /*background-image: url(../../assets/img/bg_web_l.png);*/
+  background-color: black;
   width: 100%;
   height: 100vh;
-  background-size: cover;
-  background-position: right center;
+  /*background-size: cover;
```
Comment (mikinty) [inline src/css/menu.css:188]: Also delete this

**Review by mikinty — CHANGES_REQUESTED** [review]
We have version control so can you just delete things instead of commenting them out?

**Comment by mikinty** [comment]
Also can you add some screens for mobile and desktop here? (I think video is too large so just do pictures)

**Comment by elucidium** [comment]
Screenshots (desktop views, mobile navigation menu):
Screen Shot 2020-11-02 at 10 34 38
Screen Shot 2020-11-02 at 10 34 46
Screen Shot 2020-11-02 at 10 34 52
Screen Shot 2020-11-02 at 10 34 57
Screen Shot 2020-11-02 at 10 35 21
Screen Shot 2020-11-02 at 10 35 31

**Comment by elucidium** [comment]
Resolved all (although there's still a decent amount of commented out code in CSS files I didn't modify; haven't deleted those for now)

**Review by mikinty — APPROVED** [review]
thx looks good

**Pull request merged**